deeplinkdispatch-processor/src/test/java/com/airbnb/deeplinkdispatch/DeepLinkAnnotatedElementTest.java

```diff
@@ -21,29 +21,47 @@
 
   @Test public void testValidUri() throws MalformedURLException {
     DeepLinkAnnotatedElement annotatedElement = new DeepLinkAnnotatedElement(
-        "airbnb://example.com/{foo}/bar", element, DeepLinkEntry.Type.CLASS);
+        "airbnb://example.com/{foo}/bar", element, DeepLinkEntry.Type.CLASS, null);
 
     assertThat(annotatedElement.getUri()).isEqualTo("airbnb://example.com/{foo}/bar");
   }
 
   @Test public void testQueryParam() throws MalformedURLException {
     DeepLinkAnnotatedElement annotatedElement = new DeepLinkAnnotatedElement(
-        "airbnb://classDeepLink?foo=bar", element, DeepLinkEntry.Type.CLASS);
+        "airbnb://classDeepLink?foo=bar", element, DeepLinkEntry.Type.CLASS, null);
 
     assertThat(annotatedElement.getUri()).isEqualTo("airbnb://classDeepLink?foo=bar");
   }
 
   @Test public void testInvalidUri() {
     try {
-      new DeepLinkAnnotatedElement("http", element, DeepLinkEntry.Type.CLASS);
+      new DeepLinkAnnotatedElement("http", element, DeepLinkEntry.Type.CLASS, null);
```

Comment:
add another constructor to `DeepLinkAnnotatedElement` without the last argument so you dont need to pass it everywhere even when you dont need to